### PR TITLE
[FSTORE-1440] Merge the python libraries into their subdirectories

### PR DESCRIPTION
**Pull request by aversey**
The end goal of these changes is merging of [hopsworks-api](https://github.com/logicalclocks/hopsworks-api), [feature-store-api](https://github.com/logicalclocks/feature-store-api) and [machine-learning-api](https://github.com/logicalclocks/machine-learning-api) into one project and repo and enabling management of dependencies via Python extras.

There are several PRs doing it. This is the first of them, which simply merges hsfs and hsml into their corresponding subdirectories.

The next RP: https://github.com/logicalclocks/hopsworks-api/pull/215

### CODE REVIEW

**Comment by aversey** [comment]
It was decided to split the PR into several smaller ones, so this one is now just about merging hsfs and hsml into corresponding subdirectories.

**Comment by aversey** [comment]
In this PR, `git blame` of hsml and hsfs files shows me, but this is later fixed by `git mv`.

File: hsfs/java/beam/pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kennethmhc) [inline hsfs/java/beam/pom.xml:1]: It contains java code also, is it intended?

Comment (aversey) [reply hsfs/java/beam/pom.xml:1]: Yes, Davit said to move java code too.

**Review by kennethmhc — APPROVED** [review]

**Pull request merged**